src/main/java/server/application/router/CobSpecRoutes.java

```diff
@@ -0,0 +1,86 @@
+                new FormController(resourceManager).handleGet(request, response));
+
+        post("/form", (Request request, ResponseBuilder response) ->
+                new FormController(resourceManager).handlePost(request, response));
+
+        put("/form", (Request request, ResponseBuilder response) ->
+                new FormController(resourceManager).handlePut(request, response));
+
+        delete("/form", (Request request, ResponseBuilder response) ->
+                new FormController(resourceManager).handlePost(request, response));
+
+        get("/method_options", (Request request, ResponseBuilder response) ->
+                new MethodOptionsController(resourceManager).handleGet(request, response));
+
+        post("/method_options", (Request request, ResponseBuilder response) ->
+                new MethodOptionsController(resourceManager).handlePost(request, response));
+
+        put("/method_options", (Request request, ResponseBuilder response) ->
+                new MethodOptionsController(resourceManager).handlePut(request, response));
+
+        options("/method_options", (Request request, ResponseBuilder response) ->
+                new MethodOptionsController(resourceManager).handleOptions(request, response));
+
+        head("/method_options", (Request request, ResponseBuilder response) ->
+                new MethodOptionsController(resourceManager).handleHead(request, response));
+
+        get("/file1", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
+
+        get("/file2", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
+
+        get("/image.gif", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
+
+        get("/image.jpeg", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
+
+        get("/image.png", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
+
+        get("/partial_content.txt", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
+
+        get("/patch-content.txt", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
+
+        patch("/partial_content.txt", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handlePatch(request, response));
+
+        get("/text-file.txt", (Request request, ResponseBuilder response) ->
+                new StaticFileController(resourceManager).handleGet(request, response));
```

Comment:
With all these hard-coded static file routes, I think you've lost the ability to add a file to the directory and have it served.  What you need to be able to do is to set a fallback lambda that will run when nothing else matches.  I think there is some flexibility missing here as it stands.